packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-info.tsx

```diff
@@ -40,7 +40,9 @@ const Info = observer(({ handleSelect, item, list }: TInfo) => {
     const [selected_tab, setSelectedTab] = React.useState<TSelectedTab>(TABS.DESCRIPTION);
     const contract_types: TContractType[] | undefined = getContractTypes(list, item)?.filter(
         (i: { value: TContractType['value'] }) =>
-            i.value !== 'rise_fall_equal' && i.value !== TURBOS.SHORT && i.value !== VANILLALONG.PUT
+            i.value !== TRADE_TYPES.RISE_FALL_EQUAL &&
```

Comment:
@kate-deriv nit-pick: i suggest doing destructuring here in order to reduce the number of lines :)